server.go

```diff
@@ -82,6 +82,7 @@ type service struct {
 	server interface{} // the server for service methods
 	md     map[string]*MethodDesc
 	sd     map[string]*StreamDesc
+	meta   interface{}
```

Comment:
https://talks.golang.org/2014/names.slide#4

A good name is:
Consistent (easy to guess),
Short (easy to type),
Accurate (easy to understand).